server/utils/logger.ts

```diff
@@ -131,6 +198,8 @@ export async function expressLogger(req: Request, res: Response, next: NextFunct
   req.log = log.child({
     id: req.reqId,
     user: req.user?.id,
+    timestamp: new Date().toISOString(),
```

Comment:
There's actually already a time field on each log statement.  I don't think this is required?

Reply:
Oh not now I don't think it is. Will remove.